js/additionalscripts.js

```diff
@@ -289,23 +289,23 @@ function codeButtons() {
   })
 }
 
-//a function to control a click on internal links
-function linkclick(event, that) {
-  //prevent the link from actually navigating to the url
-  event.preventDefault()
-  //grab the url to which the link is pointing
-  var url = $(that).attr("href")
-  // call the navigateContent function and pass that url to it
-  navigateContent(url)
-  //make sure the window recognizes this and adds it to the history queue for back and refresh actions
-  window.history.pushState(
-    {
-      url: url,
-    },
-    "",
-    url
-  )
-}
+// //a function to control a click on internal links
```

Comment:
Please either add a to-do to review and remove this block, or remove it entirely.